h2o-dist/index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0060)http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/index.html -->
+<html class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
This does not belong here.  There should not be any references to rel-xia in this file.